__tests__/index.test.jsx

```diff
@@ -47,6 +47,17 @@ describe("Home", () => {
     expect(ourValuesImage).toHaveAttribute("alt", "our values image");
   });
 
+  //test footer section
```

Comment:
Writing comments is good but I don't see any reason to use comments here because you have details next line with a description of the test case that tells everything.